mobile/ios/Podfile.lock

```diff
@@ -602,7 +602,7 @@ PODS:
   - BoringSSL-GRPC/Implementation (0.0.24):
     - BoringSSL-GRPC/Interface (= 0.0.24)
   - BoringSSL-GRPC/Interface (0.0.24)
-  - cloud_firestore (4.3.1):
+  - cloud_firestore (4.4.2):
```

Comment:
Hi @Mnoble-19, has the pubspec been updated??

Reply:
Hey @NoahNsimbe, yes I ran `pub upgrade`. For some dependencies, there are newer packages but not compatible with dependency constraints.